### PR TITLE
Takeshi/sample size calculator

### PR DESCRIPTION
**Pull request by ihsekat**
Basic title & content SEO optimizations for the calculator page

### CODE REVIEW

File: website/resources/sample-size-calculator/index.hbs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ---
 layout: page
+TR_seo_title: "Sample Size Calculator - Optimizely"
 TR_visible_title: A/B Test Sample Size Calculator
+description: Quickly and easily calculate the sample size needed for your A/B Testing experiments with our statistical significance calculator!
```
Comment (dtothefp) [inline website/resources/sample-size-calculator/index.hbs]: @ihsekat you will have to add a TR_ prefix to the description property. Also I'm not sure this description property is referenced in the yml. @stewsmith can you check?

**Comment by ihsekat** [comment]
@dtothefp Hey David, I tried adding the TR_ prefix to the description property, but it didn't work (see Test 2 above)

**Comment by ihsekat** [comment]
@dtothefp wrapper.hbs looks like this:

<meta name="description" content="{{#if this.description}}{{this.description}}{{else}}Deliver your best customer experiences at every touchpoint on the web and mobile apps.{{/if}}">

**Comment by dtothefp** [comment]
@KarenLoughrey could you please ingest these strings into Smartling?

**Comment by dtothefp** [comment]
retest this please

**Comment by dtothefp** [comment]
@ihsekat works for me, got here https://www.optimizelystaging.com/takeshi/sample-size-calculator/resources/sample-size-calculator/ and have a look 

screen shot 2015-04-09 at 9 50 43 am

**Comment by KarenLoughrey** [comment]
@dtothefp @ihsekat Translation in progress - should be ready on Monday. @arthuracs I will be out of office on Monday - can you please update this PR once Welocalize confirm they're in?

**Comment by arthuracs** [comment]
@dtothefp @ihsekat Welocalize confirmed that the translation are done and live

**Pull request merged**